identities/tests.py

```diff
@@ -278,6 +278,59 @@ def test_read_identity_search_multiple(self):
         data = response.json()
         self.assertEqual(len(data["results"]), 2)
 
+    def test_read_identity_search_msisdn_inactive_filter(self):
+        # Setup
+        self.make_identity(id_data={
+            "details": {
+                "name": "Test Name 2",
+                "default_addr_type": "msisdn",
+                "personnel_code": "23456",
+                "addresses": {
+                    "msisdn": {
+                        "+27123": {"default": True, "inactive": True}
+                    }
+                }
+            }
+        })
+        self.make_identity(id_data={
+            "version": 2,
+            "details": {
+                "name": "Test Name 4",
+                "addresses": {
+                    "msisdn": {
+                        "+27123": {}
+                    }
+                }
+            }
+        })
+        # Execute
+        response_include_inactive = self.client.get(
```

Comment:
Please add a query without the `include_inactive` param to validate the default setting also works